### PR TITLE
Fix facet institution

### PR DESCRIPTION
**Pull request by SammyIsConfused**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]

##### Description of change
The collection definitions within the Facet institution refer to `@description` as an attribute rather than an XML node. It appears that the schema itself is fine, so all that needed to change was these two definition XMLs. 

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: autotest/Tests/tests/facet/institution/itemdefinition/3/b2be4e8e-a0d4-4e6a-b9ff-4c65a7c8024e.xml [hunk 1]
```diff
@@ -120,8 +120,8 @@
               </title>
               <targetnodes>
                 <com.dytech.edge.wizard.TargetNode>
-                  <target>/item/</target>
-                  <attribute>description</attribute>
+                  <target>/item/description</target>
+                  <attribute></attribute>
```
Comment (SammyIsConfused) [inline autotest/Tests/tests/facet/institution/itemdefinition/3/b2be4e8e-a0d4-4e6a-b9ff-4c65a7c8024e.xml:124]: I would have shortened these to `<attribute/>` but all the other references to empty attributes in these itemdefinition XMLs use the long-form so I thought it best to keep consistent.

**Comment by edalex-ian** [comment]
Heya @SammyIsConfused 

The big issue here is that all the items have their descriptions in an attribute rather than an element. So all the items need to be updated to pull out from the attribute and put into the new element. For example, look at this: https://github.com/openequella/openEQUELLA/blob/7e6d25b6030d469e98acb06047edfb977061c356/autotest/Tests/tests/facet/institution/items/101/19045/_ITEM/item.xml

Note currently it has:

```xml
<item description="SPARC (Scalable Processor Architecture) is a reduced instruction set computing (RISC) instru...
```

Which should be:

```xml
<item><description>SPARC (Scalable Processor Architecture) is a reduced instruction set computing (RISC) instru...
```

**Comment by SammyIsConfused** [comment]
The only broken test was ItemSummaryTest in GHA New UI. Retrying.

**Comment by edalex-ian** [comment]
> The only broken test was ItemSummaryTest in GHA New UI. Retrying.

Hmmm, becoming a bit of a menace that one - failed for me too (but old UI). Did you happen to catch which test?

**Comment by SammyIsConfused** [comment]
> Hmmm, becoming a bit of a menace that one - failed for me too (but old UI). Did you happen to catch which test?

As discussed on Slack, it was this one. testAttachmentViewCount.
image

**Review by edalex-ian — APPROVED** [review]
LGTM - ran up the institution locally, and all is well. :+1:

**Comment by SammyIsConfused** [comment]
image
Argh! Another flakey test.

**Comment by mrblippy** [comment]
> image
> Argh! Another flakey test.

That test failed multiple times in a row on one of my builds yesterday

**Pull request merged**